packages/node_modules/@ciscospark/plugin-people/src/people.js

```diff
@@ -51,9 +51,13 @@ const People = SparkPlugin.extend({
    *   // => success
    */
   get(person) {
+    if (!person) {
```

Comment:
There was only one integration test, but I had actually written all the tests out before I found it, so it looks like git didn't think of it as a move.